Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -419,8 +419,9 @@ define([
             instanceScale.x = 1.0;
             instanceScale.y = 1.0;
             instanceScale.z = 1.0;
-            var scale = featureTable.getProperty('SCALE', i, ComponentDatatype.FLOAT);
+            var scale = featureTable.getProperty('SCALE', i, ComponentDatatype.FLOAT, 1);
```

Comment:
Yes. Before I do that, could you help me understand the difference between JSON and binary data? It seems that per `Cesium3DTileFeatureTable.js`:

```javascript
Cesium3DTileFeatureTable.prototype.getProperty = function(semantic, featureId, componentType, featureSize) {
        var jsonValue = this.json[semantic];
        if (defined(jsonValue)) {
            var byteOffset = jsonValue.byteOffset;
            if (defined(byteOffset)) {
                // This is a reference to the binary
                featureSize = defaultValue(featureSize, 1);
                var typedArray = this.getTypedArrayForSemantic(semantic, byteOffset, componentType, this.featuresLength, featureSize);
                var subArray = typedArray.subarray(featureId * featureSize, featureId * featureSize + featureSize);
                if (subArray.length === 1) {
                    return subArray[0];
                }
                return subArray;
            }
        }
        if (Array.isArray(jsonValue)) {
            return jsonValue.slice(featureId * featureSize, featureId * featureSize + featureSize);
        }
        return jsonValue;
    };
```

When we are dealing with binary if the element we are looking to get is a single item array, we return the element:

```javascript
if (subArray.length === 1) {
    return subArray[0];
}		
return subArray;
```

That's not enforced in the case of plain JSON:

```javascript
if (Array.isArray(jsonValue)) {
    return jsonValue.slice(featureId * featureSize, featureId * featureSize + 	featureSize);
}		
```

This seems like an fault? Perhaps it should be something like:

```javascript
if (Array.isArray(jsonValue)) {
     subArray = jsonValue.slice(featureId * featureSize, featureId * featureSize + featureSize);
      if (subArray.length === 1) {
  		return subArray[0];
      }
      return subArray;
}
return jsonValue;
```

?